src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/FluidPump.java

```diff
@@ -185,6 +217,17 @@ private Block findNextFluid(@Nonnull Block fluid) {
         return null;
     }
 
+    @Nonnull
+    private ItemStack getFilledBottle(@Nonnull Block fluid){
+        switch(fluid.getType()){
+            case WATER:
+            case BUBBLE_COLUMN:
+                return waterBottle;
```

Comment:
Fairly sure we are  going to want a `.clone()` here?

Reply:
will do, forgot to clone the object